rkt/run.go

```diff
@@ -206,25 +109,19 @@ func runRun(args []string) (exit int) {
 		stderr("run: cannot open store: %v", err)
 		return 1
 	}
-	ks := getKeystore()
 
-	s1img, err := findImage(flagStage1Image, ds, ks, false)
+	s1img, err := findImage(flagStage1Image, "", ds, nil, false)
```

Comment:
It's not a regression, we weren't verifying stage1 at all previously because local files didn't go through verification.

Let's address that in a separate PR, it will probably get more complicated because I suspect we'll want to pin the default stage1 to a specific imageid/sha so unless your running with a stage1 flag it doesn't need to verify the stage1 on every startup without `--insecure-skip-verify`.